packages/testcontainers/src/containers/DeFiDContainer.ts

```diff
@@ -35,7 +35,7 @@ export abstract class DeFiDContainer extends DockerContainer {
     if (process?.env?.DEFICHAIN_DOCKER_IMAGE !== undefined) {
       return process.env.DEFICHAIN_DOCKER_IMAGE
     }
-    return 'defi/defichain:master-2a236bb79'
+    return 'defi/defichain:master-57c099c1a'
```

Comment:
I believe this is the docker image is the latest. If not, reviewer please change it for me.

Reply:
https://github.com/DeFiCh/jellyfish/pull/1273 has the ain v2.7.0 release image. Once it get merged in. should be good to go.

Reply:
https://github.com/DeFiCh/jellyfish/pull/1273 has the ain v2.7.0 release image. Once it get merged in. should be good to go.